release/models/network-instance/openconfig-evpn.yang

```diff
@@ -40,7 +40,21 @@ module openconfig-evpn {
     domains, this is not currently supported and requires an extension
     of the model.";
 
-  oc-ext:openconfig-version "0.8.0";
+  oc-ext:openconfig-version "0.10.0";
+
+  revision "2024-06-07" {
+   description
+      "Add new local-endpoint-vnis config container to Vxlan connection point to allow
+      configuration of local VNI-to-VLAN and VNI-to-VRF maps";
+   reference   "0.10.0";
+  }
+
+  revision "2024-06-07" {
+   description
+      "Add new vni-list leaf list to EVI config to support VLAN-aware-bundle
+      MACVRFs.  Make existing vni leaf conditional on MACVRF type being VLAN-based";
+   reference   "0.9.0";
+  }
```

Comment:
Do want to merge these in the order for #1125, #1126, #1127?  If yes,  the revision statements can reflect that.

(ie: this PR #1125 should have just version 0.10.0.  Note that 0.9.0 conflicts with  recently merged version https://github.com/openconfig/public/blob/6e28490719bd8a3966ce9a78c82c3b7a5a249412/release/models/network-instance/openconfig-evpn.yang#L45-L50)

Reply:
Should be fixed now